javascript/chronometer.js

```diff
@@ -1,26 +1,68 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
+    this.msIntervalId = 0;
+    this.msTime = 0;
   }
+
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+      if (callback) { callback(); }
```

Comment:
in a single line conditional you can remove the curly brackets from the statement. Similar to writing methods in a single line.